tests/test_cln_rs.py

```diff
@@ -382,3 +382,15 @@ def test_grpc_decode(node_factory):
         string=inv.bolt11
     ))
     print(res)
+
+
+def test_rust_plugin_subscribe_wildcard(node_factory):
```

Comment:
This is great, but please use `@pytest.mark.xfail(strict=True)` annotation above this, and remove it in the next commit.  That pattern means you don't break bisect, and it's clear...